src/main/resources/messages

```diff
@@ -15,3 +15,18 @@ footer.contact.text = Contact
 footer.contact.url = https://www.gov.uk/government/organisations/hm-revenue-customs/contact
 footer.welshHelp.text = Rhestr o Wasanaethau Cymraeg
 footer.welshHelp.url = https://www.gov.uk/cymraeg
+
+addToAList.heading.0 = You have not added any items
```

Comment:
I'd be in favour of adding messages here to be more idiomatic. However, I feel we should hold off on this for now until we can implement a solution that will be consistent for all our components. What do you think?